src/dists.ad.js

```diff
@@ -517,6 +517,44 @@ var DiagCovGaussian = makeDistributionType({
   }
 });
 
+function laplaceSample(mu, b) {
+  // Generated from goo.gl/3BxCGd (wiki)
+  var z = util.random();
+  var u = (1 - z) * -0.5 + z * 0.5;
+  return mu - b * Math.sign(u) * Math.log(1 - 2 * Math.abs(u));
+}
+
+function laplaceScore(mu, b, x) {
+  'use ad';
+  return -1 * (Math.log(2 * b) + Math.abs(x - mu) / b);
+}
+
+var Laplace = makeDistributionType({
+  name: 'Laplace',
+  desc: 'Distribution over ``[-Infinity, Infinity]``',
+  params: [{name: 'mu', desc: 'mean', type: types.unboundedReal},
+           {name: 'b', desc: 'scale', type: types.positiveReal}],
+  wikipedia: true,
+  mixins: [continuousSupport],
+  sample: function() {
+    return laplaceSample(ad.value(this.params.mu), ad.value(this.params.b));
+  },
+  score: function(val) {
+    return laplaceScore(this.params.mu, this.params.b, val);
+  },
+  base: function() {
+    return new Laplace({mu: 0, b: 1});
+  },
+  transform: function(x) {
+    var mu = this.params.mu;
+    var b = this.params.b;
+    return ad.scalar.add(ad.scalar.mul(b, x), mu);
```

Comment:
> The style seems to be split among distributions (eg :517 or :615)

Yeah, I think that's just because we wrote lots of these by hand before we had the transform.